src/components/SingleResultPage/SingleResultPage.js

```diff
@@ -52,20 +54,20 @@ const SingleResultPage = ({ business, user, addFavorite, deleteFavorite }) => {
   return(
     <section className="single-result-section">
       <article className="single-result">
-        <img className="business-photo" src={img} alt={altText} />
+        <img className="business-photo" src={workingImg} alt={altText} />
         <div className="information-container">
-          <h1 className="single-title">{title}</h1>
-          <p className="address">{address}</p>
-          <p>{price}</p>
-          <div className="single-rating-container">
-            <AiFillStar/>
-            <p>{rating} / 5</p>
+          <div className="single-title-container">
+            <h1 className="single-title title-and-rating">{title}</h1>
+            <AiFillStar className="single-rating title-and-rating star-space"/>
+            <p className="single-rating title-and-rating">{rating} / 5</p>
           </div>
+          <p className="single-price">Is it expensive? {price}</p>
```

Comment:
Maybe change to this?

`<p className="single-price">Price rating: {price}</p>`